README.md

```diff
@@ -10,6 +10,7 @@ A curated list of awesome machine learning frameworks and algorithms that work o
     - [Program Synthesis and Induction](#program-synthesis-and-induction)
     - [Source Code Analysis and Language modeling](#source-code-analysis-and-language-modeling)
     - [Neural Network Architectures and Algorithms](#neural-network-architectures-and-algorithms)
+    - [Embeddings in Software Engineering](#embeddings-software-engineering)
```

Comment:
The anchoer does not work, the correct one is `embeddings-in-software-engineering`